src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -7,3 +7,46 @@ export const colors = {
   [NOT_INSTALLED]: 'gray2',
   [PENDING]: 'yellowOrange',
 };
+
+/**
+ * Integrations in the integration directory should be sorted by their popularity (weight).
+ * These weights should be hardcoded in the application itself.
+ * We can store this in a map where the key is the integration slug and the value is an integer and represents the weight.
+ * The weights should reflect the relative popularity of each integration.
+ */
+
+export const popularityWeights = {
+  // First-party-integrations
+  slack: 50,
+  github: 20,
+  jira: 10,
+  bitbucket: 10,
+  gitlab: 10,
+  pagerduty: 10,
+
+  // Plugins
+  webhooks: 20,
+  trello: 10,
+  heroku: 10,
+  pivotal: 10,
+  twilio: 10,
+  pushover: 10,
+  redmine: 10,
+  phabricator: 10,
+  opsgenie: 10,
+  teamwork: 10,
+  asana: 5,
+  victorops: 5,
+  sessionstack: 5,
+  vsts: 5,
+  segment: 1,
+  clubhouse: 1,
```

Comment:
@NisanthanNanthakumar clubhouse is actually a sentry app. Probably deserves a 5.

Reply:
@scefali it's not that popular. Is it?

Reply:
@scefali Since everything is relative, Asana has installs in the 1000s while clubhouse is in the 100s

Reply:
@NisanthanNanthakumar Again, it's not just about total number of installs. It's about what people are installing now and what we want to promote.

Reply:
@scefali true